src/city/health.c

```diff
@@ -416,10 +416,12 @@ int city_health_get_global_sickness_level(void)
     if (global_rating < LOW_SICKNESS_LEVEL && !plague_incoming) {
         global_sickness_level = SICKNESS_LEVEL_LOW;
     } else if (global_rating < MEDIUM_SICKNESS_LEVEL && !plague_incoming) {
-        global_sickness_level = SICKNESS_LEVEL_MEDIUM;
```

Comment:
I think you want something like that, remove plague_incoming code above


if (max_sickness_level == MAX_SICKNESS_LEVEL) { // one or many houses is plagued
        global_sickness_level = SICKNESS_LEVEL_PLAGUE;
    } else if (max_sickness_level >= HIGH_SICKNESS_LEVEL) { // one or many houses have sickness_level >= 90
        global_sickness_level = SICKNESS_LEVEL_HIGH;
    } else if (max_sickness_level >= MEDIUM_SICKNESS_LEVEL) { // one or many houses have sickness_level >= 60
        global_sickness_level = SICKNESS_LEVEL_MEDIUM;
    } else if (global_rating < LOW_SICKNESS_LEVEL) { // the global sickness_level is under 30
        global_sickness_level = SICKNESS_LEVEL_LOW;
    } else if (global_rating < MEDIUM_SICKNESS_LEVEL) { // the global sickness_level is under 60
        global_sickness_level = SICKNESS_LEVEL_MEDIUM;
    }